onnxruntime/core/mlas/lib/qgemm.cpp

```diff
@@ -84,120 +82,240 @@ MlasGemmU8X8ScaleSumBuffer(
     return MlasGemmU8X8ScaleSumBuffer(SumBuffer, SumBuffer, N, Scale);
 }
 
+void MLAS_QGEMM_SCALE_BIAS_OUTPUT_PROCESSOR::Process(
```

Comment:
This class could be moved to another file as it isn't directly connected to the GEMM logic around it.